week1/app.js

```diff
@@ -0,0 +1,204 @@
+var _createClass = function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; }();
```

Comment:
What's the purpose of this file? :)

Reply:
I really don't know where this file comes from it seems like a compiled class maybe vsCode have added it somehow 😄 . I just removed it 👍